app/src/components/Content/Content.styles.js

```diff
@@ -1,51 +1,153 @@
-  line-height: 70px;
-  font-size: 60px;
-  font-weight: 700;
+  @media (min-width: 350px) {
+    width: 90%;
+    font-size: 2rem;
+    line-height: 38px;
+    text-align: center;
+    margin-bottom: 10px;
+  }
+  @media screen and (min-width: 768px) {
+    margin: 3px 0;
+    width: 80%;
+    font-size: 2.5rem;
+    line-height: 2.5rem;
+    text-align: left;
+  }
+  @media (min-width: 1440px) {
+    width: 100%;
+    margin-top: 0;
+    color: ${(props) => props.theme.colors.black};
+    margin-bottom: 4px;
+    line-height: 5rem;
+    font-size: 4.5rem;
+    font-weight: 700;
+    text-align: left;
+  }
 `;
 export const StyledP = styled.p`
-  margin-top: -5px;
-  margin-bottom: 25px;
-  color: ${(props) => props.theme.colors.gray};
+  @media (min-width: 350px) {
+    color: ${(props) => props.theme.colors.gray};
+    width: 70%;
+    font-size: 1rem;
+    text-align: center;
+  }
+  @media screen and (min-width: 768px) {
+    font-size: 0.75rem;
+    width: 80%;
+    text-align: left;
+  }
+  @media (min-width: 1440px) {
+    text-align: left;
+    font-size: 1.35rem;
+    width: 32rem;
+    margin-top: -5px;
+    margin-bottom: 25px;
+    color: ${(props) => props.theme.colors.gray};
+  }
 `;
 export const StyledButton = styled.button`
-  border: none;
-  background-color: ${(props) => props.theme.colors.cyan};
-  width: 9rem;
-  height: 2.5rem;
-  border-radius: 30px;
-  color: ${(props) => props.theme.colors.white};
-  font-size: ${(props) => props.theme.fontSizes.small};
-  font-weight: 700;
+  @media (min-width: 350px) {
+    width: 10rem;
+    font-size: 1rem;
+    text-align: center;
+    margin-bottom: 170px;
+    background-color: ${(props) => props.theme.colors.cyan};
+    height: 2.5rem;
+    border-radius: 30px;
+    color: ${(props) => props.theme.colors.white};
+    font-size: ${(props) => props.theme.fontSizes.medium};
+    font-weight: 700;
+    border: none;
+  }
+  @media screen and (min-width: 768px) {
+    margin: 2px 0;
+  }
+  @media (min-width: 1440px) {
+  }
 `;
 export const StyledImgWrapper = styled.div`
-  max-height: 100%;
-  max-width: 38.33rem;
-  overflow: hidden;
+  @media screen and (min-width: 350px) {
+    margin-top: 20px;
+    width: 80%;
+    max-height: auto;
+    text-align: center;
+  }
+  @media screen and (min-width: 768px) {
+    box-sizing: border-box;
+    max-height: auto;
+    width: 55%;
+    margin-bottom: 100px;
+    margin-left: 10px;
+  }
+
+  @media screen and (min-width: 1440px) {
+    max-height: 90%;
+    width: 50%;
+    margin-bottom: 100px;
+  }
 `;
 
 export const StyledImg = styled.img`
-  padding-left: 100px;
-  max-height: 90%;
+  @media screen and (min-width: 350px) {
+    padding: 0;
+    width: 80%;
+    max-height: 110%;
+    padding-left: 0px;
+    margin: 0 auto;
+  }
+  @media screen and (min-width: 768px) {
+    max-height: auto;
+    width: 110%;
+    margin-top: 30px;
+  }
+
+  @media screen and (min-width: 1440px) {
+    max-height: auto;
+    width: 120%;
```

Comment:
I've never had to use width greater than 100%. There is probably something wrong but let's continue for now and fix that later :)

Reply:
fixed.